libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -161,7 +161,7 @@ class JSONNode {
 
 
     json_string as_string(void) const json_nothrow json_read_priority;
-    int as_int(void) const json_nothrow json_read_priority;
+    json_int_t as_int(void) const json_nothrow json_read_priority;
```

Comment:
do we need something like as_long, as_int? 
Could/should we still use int for whatever is not an offset?